### PR TITLE
adding ds-914.cr.cnaf.infn.it to egi

### PR DESCRIPTION
**Pull request by biozit**
adding ds-914.cr.cnaf.infn.it to egi

### CODE REVIEW

File: etc/cvmfs/osgstorage-auth.conf [hunk 1]
```diff
@@ -18,6 +18,7 @@ CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://daejeon-kreonet-net.nationalrese
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.jack.nrp.internet2.edu:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osdf-cache.sprace.org.br:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://mghpcc-cache.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ds-914.cr.cnaf.infn.it:8443/"
```
Comment (DrDaveD) [inline etc/cvmfs/osgstorage-auth.conf:21]: Please move this down to the list not used by OSG

**Review by DrDaveD — CHANGES_REQUESTED** [review]

File: etc/cvmfs/osgstorage-auth.conf (hunk 1 above)
Comment (biozit) [reply etc/cvmfs/osgstorage-auth.conf]: done

Comment (DrDaveD) [reply etc/cvmfs/osgstorage-auth.conf]: I'm sorry!  It is actually listed in the Europe section of osg.  So you had it in the right section, although not in the right order.  In fact, it should have been in the egi branch already but #185 and #192 have never been completed.  Let's do this right and finish those instead.

**Comment by DrDaveD** [comment]
Closing this PR.  To be replaced by another that syncs up all the osgstorage servers in the egi branch with the osg branch.

File: etc/cvmfs/osgstorage-auth.conf (hunk 1 above)
Comment (DrDaveD) [reply etc/cvmfs/osgstorage-auth.conf]: The first stage of reorganizing the osg branch was done in #197 but the other branches haven't yet been done.

**Pull request closed without merging**